### DISCUSSION TITLE
Grove - Temperature & Humidity Sensor(SHT4x) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1217] [Q&A]
# Grove - Temperature & Humidity Sensor(SHT4x) | Seeed Studio Wiki

Grove - Temperature & Humidity Sensor(SHT4x)

https://wiki.seeedstudio.com/Grove-SHT4x/

**Comment by kornpow** [comment]
Are the Kicad files available for this board?
